### ISSUE TITLE
Wtyczka tvp: nowsze (niektóre) odcinki wymagają zalogowania, podczas gdy na stronie działają bez zalogowania

### ISSUE THREAD

**Issue by pogarek** [mbebe/blomqvist#872]
Na przykład  "Na dobre i na złe", odcinek z wczoraj (z 16.11; https://vod.tvp.pl/seriale,18/na-dobre-i-na-zle-odcinki,273721/odcinek-861,S08E861,378379 ).

Instalację mam na apple tv , tam nie ma innej, łatwej,  opcji na dodanie wtyczki niż url do repozytorium, więc nie mogę użyć wtyczki z #861

**Comment by mtr81** [comment]
O której wtyczce piszesz?

**Comment by pogarek** [comment]
tvp.pl , wersja 2.2.49

**Comment by pogarek** [comment]
AppleTV to, trochę, zamknięty system, tam się nie da nic dorzucić. Wszystkie wtyczki instaluję z zipów na repozytoriach. Żeby dać "samodzielny" plik zip, to musiałbym sobie jakiś serwerek http postawić lub smb.. Byłbym wdzięczny za dodanie wtyczki z #861 do repozytorium na https://mbebe.github.io/blomqvist/.

**Comment by xxcriticxx** [comment]
@pogarek jak ty za instalowales kodi na apple tv? jaka wersje?

**Comment by pogarek** [comment]
> @pogarek czyli co 7 dni musisz wgrywac nowa wersje

Nie, mam płatne konto więc raz na rok.

**Comment by xxcriticxx** [comment]
@pogarek dzieki za informacje sorry za hijack tego posta

btw 2022 apple tv 4k jest super!

**Comment by zuzia-dev** [comment]
Wtyczka @mbebe co prawda wymaga modyfikacji, ale można sobie obejrzeć bez logowania i płacenia za cokolwiek między innymi serial "Erynie" w 4K.

**Comment by mtr81** [comment]
> Wtyczka @mbebe co prawda wymaga modyfikacji, ale można sobie obejrzeć bez logowania i płacenia za cokolwiek między innymi serial "Erynie" w 4K.

Jak to koresponduje z pierwszym postem?

**Comment by pogarek** [comment]
Jak tylko będę miał jak to sprawdzę.

Tak są obsuwy.

Czy istnieje szansa na zastąpienie starej wtyczki nową? (W repozytorium). Lub dodanie nowej apki obok starej.

**Comment by zuzia-dev** [comment]
@mtr81  zgadza się, wspomniane "Na dobre i na złe", odcinek z 16.11 na ten moment jak najbardziej działa w wersji wtyczki 2.2.49, bez logowania. Inne też działają, sprawdziłam wyrywkowo.

**Comment by pogarek** [comment]
Nie działało w momencie tworzenia tego "issue". Możliwe że inne, najnowsze odcinki , innych, audycji mają podobny problem.
